server/src/routes/api/v1/index.test.js

```diff
@@ -1,7 +1,82 @@
-// TODO
+import request from 'supertest';
+import app from '../../../configs/server';
 
-describe('api/v1 tests', () => {
-	test('always pass', () => {
-		expect(true).toBe(true);
+describe('/api/v1 routes', () => {
+  let server;
+  const application = app();
+  beforeEach(() => {
+    server = application.create();
+    jest.resetModules();
+  });
+
+  describe('/auth routes', () => {
+    describe('POST /signin', () => {
+      it('should return 200 for correct credentials', async () => {
+        const res = await request(server).post('/api/v1/auth/signin');
+        expect(res.status).toBe(200);
+      });
+    });
+
+    describe('POST /signup', () => {
+      it('should return 201 for correct inputs', async () => {
+        const res = await request(server).post('/api/v1/auth/signup');
+        expect(res.status).toBe(201);
+      });
+    });
+
+    describe('POST /forgot-password', () => {
+      it('should return 200 for correct email id', async () => {
+        const res = await request(server).post('/api/v1/auth/forgot-password');
+        expect(res.status).toBe(200);
+      });
+    });
+  });
+
+  describe('/categories', () => {
+    describe('GET /', () => {
+      it('should return 200', async () => {
+        const res = await request(server).get('/api/v1/categories');
+        expect(res.status).toBe(200);
+      });
+    });
+  });
+
+  describe('/user routes', () => {
+    describe('GET /:userID', () => {
```

Comment:
ID or Id?
Which convention should we be following?

Reply:
> userId

Yes, userId